speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -305,7 +306,7 @@ static void SynchronizeControls()
 
     std::string fileName = m_blackBoxFilePath;
     std::string buttonText = FindBlackBoxButtonTextFromPath(fileName);
-    GfuiButtonSetText(s_scrHandle, m_blackBoxButton, buttonText.c_str());
+    GfuiButtonSetText(s_scrHandle, m_blackBoxButton, MSG_BLACK_BOX_PATH_TOO_LONG/*buttonText.c_str()*/);
```

Comment:
Don't think you meant to keep this in, since it will now always show the error message on startup.

Reply:
Indeed